src/userscripts/linkHarmonyReleaseActions.user.js

```diff
@@ -0,0 +1,315 @@
+const DEFAULT_ICON_MARGIN_LEFT = "5px";
+const DEFAULT_HARMONY_ICON_URL =
+  "https://harmony.pulsewidth.org.uk/favicon.svg";
+const DEFAULT_HARMONY_BASE_URL =
+  "https://harmony.pulsewidth.org.uk/release/actions?release_mbid=";
+
+// --- Key names for GM storage ---
+const KEY_ENABLE_RG = "harmonyEnableRG";
+const KEY_ENABLE_ARTIST = "harmonyEnableArtist";
+const KEY_DIGITAL_ONLY = "harmonyDigitalOnly";
+const KEY_ICON_SIZE = "harmonyIconSize";
+const KEY_ICON_MARGIN = "harmonyIconMarginLeft";
+
+// --- CSS Class Names ---
+const CSS_LINK_CLASS = "harmony-userscript-link";
+const CSS_ICON_CLASS = "harmony-userscript-icon";
+// --- Common Selectors ---
+const releaseTableSelector = "table.tbl.mergeable-table tbody";
+const releaseTitleLinkSelector = 'a[href*="/release/"] > bdi';
+
+/**
+ * Injects CSS rules into the document's head.
+ * Prevents duplicate injection using an ID.
+ * @param {string} css - The CSS rules to inject.
+ * @param {string} id - An ID for the style element to prevent duplicates.
+ */
+function injectStylesheet(css, id) {
+  const styleId = `harmony-userscript-style-${id}`;
+  if (document.getElementById(styleId)) return;
+  const style = document.createElement("style");
+  style.id = styleId;
+  style.textContent = css;
+  document.head.appendChild(style);
+}
+
+/**
+ * Fetches a configuration value from GM storage, using a default if not found.
+ * @param {string} key - The key for GM_getValue.
+ * @param {any} defaultValue - The default value to return if the key is not found.
+ * @returns {Promise<any>} - The retrieved or default value.
+ */
+async function getConfigValue(key, defaultValue) {
+  try {
+    let value = await GM_getValue(key, defaultValue);
+    if (typeof defaultValue === "boolean") {
+      value = Boolean(value);
+    }
+    if (typeof value !== typeof defaultValue && defaultValue !== undefined) {
+      console.warn(
+        `Harmony Link Script: Config value for '${key}' has unexpected type. Using default.`
+      );
+      return defaultValue;
+    }
+    return value;
+  } catch (e) {
+    console.error(
+      `Harmony Link Script: Error getting config value for '${key}'. Using default.`,
+      e
+    );
+    return defaultValue;
+  }
+}
+
+/**
+ * Creates the Harmony link anchor (<a>) element with the icon (<img>).
+ * @param {string} mbid - The MusicBrainz Release MBID.
+ * @returns {HTMLAnchorElement|null} The created anchor element or null if mbid is invalid.
+ */
+function createHarmonyLinkElement(mbid) {
+  if (!mbid || typeof mbid !== "string" || !/^[a-f0-9\-]{36}$/.test(mbid)) {
+    console.warn(
+      "Harmony Link Script: Invalid MBID passed to createHarmonyLinkElement:",
+      mbid
+    );
+    return null;
+  }
+
+  const harmonyLink = document.createElement("a");
+  harmonyLink.href = `${DEFAULT_HARMONY_BASE_URL}${mbid}`;
+  harmonyLink.target = "_blank";
+  harmonyLink.rel = "noopener noreferrer";
+  harmonyLink.title = `View Harmony Actions for ${mbid} (opens in new tab)`;
```

Comment:
```suggestion
  harmonyLink.title = `View Harmony Release Actions (opens in new tab)`;
```
I don't think it is necessary to include the MBID in the tooltip, it is not important for the user (and in case it is, the full URL is usually shown in the browser's status bar/tooltip).